force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,23 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String helloSF = 'Hello, Salesforce!';
+        // Get the length of the String
+        String listLen = helloSF.length().toString();
         // Get a substring of the String of the first 5 characters
+        String string5 = helloSF.substring(0, 5);
         // Convert the String to lower case
-        
+        String stringValLower = helloSF.toLowerCase();
         // Return the list with the string data
+        List<String> stringList = new List<String>();
         // In the first position add the string length
         // In the second position add the substring
         // In the third position add the string in lower case
+        stringList.add(listLen);
+        stringList.add(string5);
+        stringList.add(stringValLower);
 
-        return null; // Replace null with the variable you used to store the result
+        return stringList; // Replace null with the variable you used to store the result
```

Comment:
Something I encourage new developers to think about is. "do I really need a variable here" in this case you could the the results of the String method calls directly to the List and, I think, not lose anything in readability.
```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```